pkg/plugins/generic/generic_plugin.go

```diff
@@ -197,3 +203,24 @@ func isCommandNotFound(err error) bool {
 	}
 	return false
 }
+
+func handleSriovModeStateChange(iface *sriovnetworkv1.Interface, old *sriovnetworkv1.SriovNetworkNodeState) (needDrain bool, err error) {
+	err = nil
+	needDrain = false
+	if iface.SriovMode == "" {
+		if iface.SriovMode != "legacy" && iface.SriovMode != "switchdev" {
+			return false, fmt.Errorf("generic-plugin OnNodeStateChange(): not supported SR-IOV mode %s for "+
+				"device %s, supported modes [legacy, switchdev]", iface.PciAddress, iface.SriovMode)
```

Comment:
can we have consts defined for SriovMode ? you are using "legacy" and "switchdev" strings in several places

Reply:
it only used once as string in the if condition on line 211 and in the error message in error message

Reply:
nevertheless i think its best practice to have this defined, also devlink relies on this string, it would lower the chances of errors in the future.